app/components/primer/button_component.html.erb

```diff
@@ -1,4 +1,12 @@
-<%= render Primer::Beta::BaseButton.new(**@system_arguments) do -%>
-  <%= leading_visual %><%= trimmed_content %><%= trailing_visual %><%= primer_octicon("triangle-down", ml: 2, mr: -1) if @dropdown %>
-  <%= tooltip %>
-<% end -%>
+<% if tooltip.present? %>
+  <%= render Primer::BaseComponent.new(tag: :span, position: :relative) do %>
```

Comment:
should we set the display to [inline-block](https://github.com/primer/css/blob/main/src/buttons/button.scss#L6) to preserve what .btn usually renders as? maybe`tag: :div` and `display: :inline-block`. 

similar comment for `IconButton`